doc/man3/EVP_EncryptInit.pod

```diff
@@ -388,8 +388,11 @@ of data. The amount of data written depends on the block alignment of the
 encrypted data.
 For most ciphers and modes, the amount of data written can be anything
 from zero bytes to (inl + cipher_block_size - 1) bytes.
-For wrap cipher modes, the amount of data written can be anything
-from zero bytes to (inl + cipher_block_size) bytes.
+
+When used for wrapping algorithms such as "AES-128-WRAP" and "AES-192-WRAP-PAD"
+the encrypted data buffer I<out> should have sufficent room for
+8 + (I<inl> padded to an 8 byte boundary) bytes.
+
```

Comment:
Well.. the original text already says that there might be written more than inl.